server/index.js

```diff
@@ -4,13 +4,11 @@ let app = express();
 app.use(express.static(__dirname + '/../client/dist'));
 
 app.post('/', function (req, res) {
```

Comment:
You shouldn't need a post function here - I recommend you delete it.

All we need to do with our server is `GET`. `database/seed.js` is where you are adding to your database, which would be the typical `POST`-type function.

You will need to add a second `GET` endpoint for `/shop/:productId`, but I know you already know this and it's the next step.  just what came to mind when seeing this!